### PR TITLE
document file names used in external certificate configuration

### PR DESCRIPTION
**Pull request by elevran**
document expected secret and certificates names when plugging in external CA.
This was the source of some confusion, especially in multicluster setup (e.g., Citadel certificate and key  file names for `cluster-01` can't be `cluster-01-cert.pem`, `cluster-01-key.pem`...).

/assign @frankbu

### CODE REVIEW

File: content/docs/tasks/security/plugin-ca-cert/index.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.
 
 These files are ready to use in the `samples/certs/` directory.
 
+  {{< tip >}}
+  The default Citadel installation mounts the `cacert` secret as a volume under `/etc/certs`.
+  It is configured via [command line options](/docs/reference/commands/istio_ca/index.html)
```
Comment (frankbu) [inline content/docs/tasks/security/plugin-ca-cert/index.md]: This sentence is a little confusing because if you click on the link and look at the command line options, the default is ``. Maybe rewording slightly would be better?

The default installation sets [command line options](/docs/reference/commands/istio_ca/index.html)
to configure the location of certificates and keys to the predefined file names used ...

Comment (elevran) [reply content/docs/tasks/security/plugin-ca-cert/index.md]: rephrased to clarify. PTAL and lgtm if acceptable

File: content/docs/tasks/security/plugin-ca-cert/index.md [hunk 2]
```diff
@@ -43,11 +43,7 @@ Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.
 These files are ready to use in the `samples/certs/` directory.
 
   {{< tip >}}
-  The default Citadel installation mounts the `cacert` secret as a volume under `/etc/certs`.
-  It is configured via [command line options](/docs/reference/commands/istio_ca/index.html)
-  to load certificates and keys from the predefined file names used in the command below
-  (i.e., root certificate must be in a file named `root-cert.pem`, Citadel key in
-  `ca-key.pem`, etc.)
+  The default Citadel installation sets [command line options](/docs/reference/commands/istio_ca/index.html) to configure the location of certificates and keys based on the predefined secret and file names used in the command below (i.e., secret named `cacert`,root certificate in a file named `root-cert.pem`, Citadel key in `ca-key.pem`, etc.)
```
Comment (frankbu) [inline content/docs/tasks/security/plugin-ca-cert/index.md]: missing space (" ") character after comma: `cacert`,root certificate ...

Also one more thought, maybe it would be good to add another sentence, something like this:

You must use these specific file names for your certificates and keys unless you reconfigure Citadel when you deploy it.

Comment (elevran) [reply content/docs/tasks/security/plugin-ca-cert/index.md]: updated: `You must use these specific secret and file names, or reconfigure Citadel when you deploy it.`
